pacman_contest_cluster.py

```diff
@@ -1031,9 +1073,9 @@ def _load_teams(team_names_file):
     runner = ContestRunner(**settings)  # Setup ContestRunner    
     
     if resume_competition_folder != '':
-        runner.resume_contest_remotely(hosts, resume_competition_folder)  # Now run ContestRunner with the hosts!
+        runner.resume_contest_remotely(hosts, resume_competition_folder, settings['staff_teams_vs_others_only'])  # Now run ContestRunner with the hosts!
```

Comment:
Why do we pass this particular option to the method? We already passing staff_teams_vs_others_only when we stup the ContestRunner so we can save it there, as we do with all the other options right?